### ISSUE TITLE
ZOD Error: Expected number, received string

### ISSUE THREAD

**Issue by patrickbenkoe** [toebeann/gib#17]
Hey,

i am running into a Problem, which is included here :-(

````
159 |     const json = JSON.stringify(obj, null, 2);
160 |     return json.replace(/"([^"]+)":/g, "$1:");
161 | };
162 | class ZodError extends Error {
163 |     constructor(issues) {
164 |         super();
              ^
ZodError: [
  {
    "code": "invalid_type",
    "expected": "number",
    "received": "string",
    "path": [
      "UserLocalConfigStore",
      "Software",
      "Valve",
      "Steam",
      "apps",
      "264710",
      "BadgeData"
    ],
    "message": "Expected number, received string"
  }
]
 errors: [
  {
    "code": "invalid_type",
    "expected": "number",
    "received": "string",
    "path": [
      "UserLocalConfigStore",
      "Software",
      "Valve",
      "Steam",
      "apps",
      "264710",
      "BadgeData"
    ],
    "message": "Expected number, received string"
  }
]

      at new ZodError (/private/tmp/bunx-501-@toebeann/gib/node_modules/zod/lib/index.mjs:164:9)
      at error (/private/tmp/bunx-501-@toebeann/gib/node_modules/zod/lib/index.mjs:587:31)
      at parse (/private/tmp/bunx-501-@toebeann/gib/node_modules/zod/lib/index.mjs:692:15)

Bun v1.1.26 (macOS arm64)
```
`

**Comment by toebeann** [comment]
Thanks for letting me know, I'll see what I can do

**Comment by toebeann** [comment]
Hi @patrickbenkoe, when you are able can you try again and let me know if it's fixed with [v0.7.3](https://github.com/toebeann/gib/releases/tag/v0.7.3)?

Just run the standard command again:
```bash
curl -fsSL https://cdn.jsdelivr.net/gh/toebeann/gib/gib.sh | bash
```

**Comment by patrickbenkoe** [comment]
> Hi @patrickbenkoe, when you are able can you try again and let me know if it's fixed with [v0.7.3](https://github.com/toebeann/gib/releases/tag/v0.7.3)?
> 
> Just run the standard command again:
> 
> ```shell
> curl -fsSL https://cdn.jsdelivr.net/gh/toebeann/gib/gib.sh | bash
> ```

works fine again! what a fast support! wow
`- tobey ♥`
